extensions/sql-database-projects/src/controllers/projectController.ts

```diff
@@ -671,28 +672,30 @@ export class ProjectsController {
 	 * Imports a new SQL database project from the existing database,
 	 * prompting the user for a name, file path location and extract target
 	 */
-	public async importNewDatabaseProject(context: azdata.IConnectionProfile | any): Promise<void> {
+	public async importNewDatabaseProject(context: azdata.IConnectionProfile | any): Promise<CreateProjectFromDatabaseDialog> {
+		let profile = this.getConnectionProfileFromContext(context);
```

Comment:
Use `const` instead of `let` if a variable shouldn't be set again in its lifetime